selectrum.el

```diff
@@ -278,6 +278,43 @@ This option is a workaround for 2 problems:
   wrapping."
   :type 'integer)
 
+(defcustom selectrum-multiline-display-settings
+  '((match      "->"  success)
+    (truncation "..." shadow)
+    (newline    "\\n" warning)
+    (whitespace ".."  shadow))
+  "Indicators used to represent formatting in multi-line candidates.
```

Comment:
Similar to the changelog could we restructure this phrase a bit? Maybe: Settings to configure formatting of multi-line candidates.